autoload/which_key/view.vim

```diff
@@ -26,17 +26,21 @@ function! s:calc_layout(mappings) abort " {{{
   let smap = filter(copy(a:mappings), 'v:key !=# "name" && !(type(v:val) == s:TYPE.list && v:val[1] == "which_key_ignore")')
   let layout.n_items = len(smap)
   let displaynames = which_key#view#get_displaynames()
-  let length = values(map(smap,
-        \ 'strdisplaywidth(get(displaynames, toupper(v:key), v:key).'.
-        \ '(type(v:val) == s:TYPE.dict ? get(v:val, "name", "") : v:val[1]))'))
 
-  let maxlength = max(length) + g:which_key_hspace
+  let prefix_length = values(map(copy(smap),
+        \ 'strdisplaywidth(get(displaynames, toupper(v:key), v:key))'))
+  let suffix_length = values(map(smap, 'strdisplaywidth('.
+        \ 'type(v:val) ==s:TYPE.dict ? get(v:val, "name", "") : v:val[1])'))
```

Comment:
```suggestion
let suffix_length = values(map(smap,
        \ 'strdisplaywidth(type(v:val) ==s:TYPE.dict ? get(v:val, "name", "") : v:val[1])'))
```

Reply:
The line ends up being a bit long, though (92 chars). How about
```suggestion
  let suffix_length = values(map(smap,
        \ 'strdisplaywidth(type(v:val) ==s'.
        \ ':TYPE.dict ? get(v:val, "name", "") : v:val[1])'))
```

Reply:
Ok.